panther_driver/src/driver_node.py

```diff
@@ -385,8 +434,9 @@ def driverNode():
                 br.sendTransform(tf)
 
             if publish_odometry == True:
+                odom_msg.header.stamp = rospy.Time.now()
                 odom_msg.header.frame_id = odom_frame
-                odom_msg.header.stamp = now
+                odom_msg.header.seq = seq_counter
```

Comment:
do we need to set `seq`?

Reply:
There is a place for that in the message so why not

Reply:
AFAIK it isn't really used and it just adds extra counter to our code, so I suggest removing it